shell/src/main/java/alluxio/cli/LogLevel.java

```diff
@@ -196,6 +196,7 @@ public static void logLevel(String[] args, AlluxioConfiguration alluxioConf)
           targetInfoList.add(jobWorker);
         }
       } else if (target.contains(":")) {
+        // TODO(jiacheng): Support all other roles, not just worker
```

Comment:
The original command only considers the manually specified targets to be a worker. I think this should support all possible roles, not limited to workers and job-workers. The change will not be trivial so I intend to do it in another PR, to keep the review cycle short.

Reply:
Ongoing changes can be found in https://github.com/Alluxio/alluxio/pull/12874/files